components/ProductDetail/ProductBidDrawer.tsx

```diff
@@ -14,22 +14,27 @@ import {
   useToast,
 } from '@chakra-ui/react';
 import { useRouter } from 'next/router';
-import { ChangeEvent, useState } from 'react';
+import { ChangeEvent, useEffect, useState } from 'react';
 
 import { bidAPI } from 'apis';
 import useForm from 'hooks/useForm';
-import { biddingPriceValidation, priceFormat } from 'utils';
+import { biddingPriceValidation, priceFormat, setToastInfo } from 'utils';
 
+interface BidUser {
+  biddingPrice?: number;
+}
 interface ProductBidDrawerProps {
   minimumPrice: number;
   isOpen: boolean;
   onClose: () => void;
+  bidder: BidUser;
 }
```

Comment:
```suggestion
interface ProductBidDrawerProps {
  minimumPrice: number;
  isOpen: boolean;
  onClose: () => void;
  bidder: {
    biddingPrice?: number;
  };
}
```
혹시 한번에 작성하는 것은 어떠신가요? 속성도 1개여서 가독성이 많이 나빠지지 않을것 같습니다!
수경님을 포함하여 다른분들은 어떻게 생각하시나요??

Reply:
아래 리뷰와는 별개로 이 컴포넌트에서는 같이 사용해도 좋을 것 같아요,,!